src/Symfony/Component/Routing/CHANGELOG.md

```diff
@@ -32,7 +32,7 @@ CHANGELOG
 
    ```php
    $route = new Route();
-   $route->setPattern('/article/{id}');
+   $route->setPath('/article/{id}');
```

Comment:
this is wrong. It is the example of the code **before** the change, so it is logical to use the old name here